newrelic-nri-kube-events.yaml

```diff
@@ -1,6 +1,6 @@
 package:
   name: newrelic-nri-kube-events
-  version: 2.3.0
+  version: 2.2.7
```

Comment:
```suggestion
  version: 2.2.9
```

Reply:
I thought I'd just do a simple revert and let the bot update it back to the latest version.